### PR TITLE
Update on-chain interoperability endpoints

### LINKED ISSUE #1324: Update on-chain interoperability endpoints
### Description

Update the following endpoint implementation `/api/v3/blockchain/apps` to reflect the below-mentioned criteria.

### Acceptance Criteria

- `chainID` request query param has CSV support
- `isDefault` request query param is not supported anymore
- `isDefault` is removed from the response body
  - Related config entries are cleared
- Integration tests are up-to-date
- Swagger is up-to-date

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?
This PR resolves #1324

### How was it solved?
- [x] chainID request query param has CSV support. Already done.
- [x] isDefault request query param is not supported anymore
- [x] isDefault is removed from the response body
    - [x] Related config entries are cleared. Being used other places
- [x] Integration tests are up-to-date
- [x] Swagger is up-to-date

### How was it tested?
- [x] Local. Add few rows in `blockchain_apps` table and query

### CODE REVIEW

File: services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json [hunk 1]
```diff
@@ -178,8 +173,8 @@
             },
             "chainID": {
                 "type": "string",
-                "example": "00000001",
-                "description": "Chain ID as registered on the network"
+                "example": "00000000",
+                "description": "Chain ID of the blockchain application"
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json:177]:
```suggestion
                "description": "Chain ID of the blockchain application."
```

File: services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json [hunk 2]
```diff
@@ -323,8 +318,8 @@
         "properties": {
             "chainID": {
                 "type": "string",
-                "example": "00000001",
-                "description": "Chain ID as registered on the network"
+                "example": "00000000",
+                "description": "Chain ID of the blockchain application"
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json:322]:
```suggestion
                "description": "Chain ID of the blockchain application."
```

File: services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json [hunk 3]
```diff
@@ -24,19 +24,11 @@
...
-        "type": "string",
-        "minLength": 1
-    },
-    "isDefault": {
-        "name": "isDefault",
-        "in": "query",
-        "description": "Filter default blockchain application.",
-        "type": "boolean",
-        "minLength": 1
+        "description": "State of the blockchain application (can be expressed as CSV. Accepted values: registered,active,terminated).",
```
Comment (sameersubudhi) [inline services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json:30]: Please test the change and adjust if the recommendation is incorrect.
```suggestion
        "description": "State of the blockchain application. Can be expressed as CSV.\nAccepted values: 'registered', 'active', 'terminated'.",
```

File: tests/schemas/api_v3/blockchainAppsSchema.schema.js [hunk 4]
```diff
@@ -35,9 +35,8 @@ const blockchainAppSchema = {
 	chainID: Joi.number().integer().min(1).required(),
 	state: Joi.string().valid(...validStatuses).required(),
 	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
-	isDefault: Joi.boolean().required(),
-	lastCertificateHeight: Joi.string().required(),
-	lastUpdated: Joi.string().required(),
+	lastCertificateHeight: Joi.number().integer().min(0).required(),
+	lastUpdated: Joi.number().integer().positive().required(),
```
Comment (sameersubudhi) [inline tests/schemas/api_v3/blockchainAppsSchema.schema.js:39]: Can we also set a max limit on `lastUpdated`? It should always be less than the current time. You should be able to find similar implementation in some other schemas.

File: services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js [hunk 5]
```diff
@@ -74,27 +72,7 @@ const getBlockchainApps = async (params) => {
...
-			}
-			return appInfo;
-		},
-		{ concurrency: response.length },
-	);
-
-	blockchainAppsInfo.data = 'isDefault' in params
-		? blockchainAppsInfo.data
-		: blockchainAppsInfo.data.sort((a, b) => b.isDefault - a.isDefault);
+	blockchainAppsInfo.data = response;
```
[the remark this answers is not in the document]
Comment (priojeetpriyom) [reply services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js]: Applied

File: services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json (hunk 3 above)
Comment (priojeetpriyom) [reply services/gateway/apis/http-version3/swagger/parameters/blockchainApps.json]: Applied.

File: tests/schemas/api_v3/blockchainAppsSchema.schema.js (hunk 4 above)
Comment (priojeetpriyom) [reply tests/schemas/api_v3/blockchainAppsSchema.schema.js]: Applied.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

**Pull request merged**